FILE packages/website/src/routes/index.tsx
RANGE {"path":"packages/website/src/routes/index.tsx","old_lines":[316,320],"new_lines":[316,320],"diff_block_lines":[5,11],"hunk_header":"@@ -316,5 +316,5 @@ function FAQ() {","substantive_changed_lines":2}
@@ -316,5 +316,5 @@ function FAQ() {
         </FAQItem>
         <FAQItem question="What does Paseo mean?">
-          Stroll, in Spanish. 🚶
+          Stroll, in Spanish. 🚶‍♂️
         </FAQItem>
       </div>
